data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
On Jun 11, 2016 9:08 AM, "mugling" <notifications@github.com> wrote:
>
> In data/json/items/gun/nail.json:
>
> > @@ -1,5 +1,29 @@
> >  [
> >    {
> > +    "id": "coilgun",
> > +    "copy-from": "gun_base",
> > +    "type": "GUN",
> > +    "name": "coilgun",
> > +    "description": "A homemade gun, using electromagnets to accelerate
a ferromagnetic projectile to high velocity.  Powered by UPS.",
> > +    "weight": 3341,
> > +    "volume": 11,
> > +    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and
silent - people will want this thing.",
> > +    "bashing": 10,
> > +    "to_hit": -2,
> > +    "material": "steel",
> > +    "skill": "rifle",
> > +    "ammo": "nail",
> > +    "ranged_damage": 8,
>
> Needs much better stats to be relevant
> versus the nail rifle considering it's difficult to
> craft, requires UPS charges and is semi-auto
> only.

Please stop repeating this rationale. Weapons do not need to be "relevant"
or "competitive" versus other weapons, and that is not a legitimate
rationale for buffing arbitrary stats as you're doing here. In particular,
the sight dispersion was set specifically because it is not a factory made
wrapon, so the sights are not going to be as good. This needs to be
reverted.